File: sdks/thunkmetrc-py/wrapper/src/thunkmetrc/wrapper/models/plant_batches_waste.py
from typing import TypedDict, List, Optional, Any, Generic, TypeVar, TYPE_CHECKING
from typing_extensions import TypedDict

if TYPE_CHECKING:
    from . import *

class PlantBatchesWaste(TypedDict, total=False):
    PlantBatchId: int
    PlantBatchName: str
    PlantCount: int
    PlantWasteNumber: str
    WasteDate: str
    WasteMethodName: str
    WasteReasonName: str
    WasteUnitOfMeasureName: str
    WasteWeight: float

File: sdks/thunkmetrc-java/wrapper/src/main/java/io/github/thunkier/thunkmetrc/wrapper/models/PlantBatchesWaste.java
package io.github.thunkier.thunkmetrc.wrapper.models;
import com.fasterxml.jackson.annotation.JsonProperty;public class PlantBatchesWaste {
    @JsonProperty("PlantBatchId")
    public Integer plantBatchId;
    @JsonProperty("PlantBatchName")
    public String plantBatchName;
    @JsonProperty("PlantCount")
    public Integer plantCount;
    @JsonProperty("PlantWasteNumber")
    public String plantWasteNumber;
    @JsonProperty("WasteDate")
    public String wasteDate;
    @JsonProperty("WasteMethodName")
    public String wasteMethodName;
    @JsonProperty("WasteReasonName")
    public String wasteReasonName;
    @JsonProperty("WasteUnitOfMeasureName")
    public String wasteUnitOfMeasureName;
    @JsonProperty("WasteWeight")
    public Double wasteWeight;
}
